primefaces-selenium/primefaces-selenium-core/src/main/java/org/primefaces/selenium/internal/DefaultWebDriverAdapter.java

```diff
@@ -87,7 +87,7 @@ public WebDriver createWebDriver() {
 
         switch (config.getWebdriverBrowser()) {
             case "firefox":
-                FirefoxOptions firefoxOptions = new FirefoxOptions();
+                FirefoxOptions firefoxOptions = new FirefoxOptions().configureFromEnv();
```

Comment:
whats that? please create a seperate issue + PR for this

Reply:
Done